System.IO.FileSystem/NativeIO.cs

```diff
@@ -0,0 +1,53 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.IO
+{
+    internal static class NativeIO
+    {
+        internal const string FSRoot = @"\";
+
+        internal const uint EmptyAttribute = 0xFFFFFFFF;
+
+        // all these values are from FS_decl.h
+        internal const int FSMaxPathLength = 260 - 2;
+        internal const int FSMaxFilenameLength = 256;
+        internal const int FSNameMaxLength = 7 + 1;
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
```

Comment:
Does the Native format allows for formatting internal drive (i:)?

Reply:
yes!

Reply:
Perhaps `FormatDrive` would be a better name for this function, as `Format` can mean many things (e.g. `String.Format`)

Reply:
Not sure about this one... 🤔 
Format is a widely used term for _format_ so... 
_RE_format makes it sound unfamiliar with the usual file APIs: FATFS, littlefs, etc.
Not sure if it's worth the potential 🤨 reaction.

Reply:
`RepairDrive` or `WipeDrive` or `CleanFileSystem`, but I think you have now removed this from the managed lib, looking at the latest checkin...

Reply:
This is in the NativeIO interface class.

At the native interop (with the current file systems we have on the works) this ends up calling format & mount in both FATFS and littlefs...

Even in Windows, when one wants to repair or format a disk, the menu command is called format. Why changing this when it's so commonly used and the intention is perfectly clear and adopted? 😁